### DISCUSSION TITLE
Is installing `entrez-direct` via `conda` equivalent to installing it via .sh scripts?

### DISCUSSION THREAD

**Discussion by TullyMonster** [biostars/biostar-handbook-issues#304] [General]
Method 1:
```bash
yes no | sh -c "$(curl -fsSL ftp://ftp.ncbi.nlm.nih.gov/entrez/entrezdirect/install-edirect.sh)" > /dev/null
```

Method 2:
```bash
conda install -c bioconda entrez-direct
```

Q: Are the two methods equivalent? Which is more recommended?

I think it's the second kind. Because this only requires the user to activate the virtual environment before use without the configuration of environment variables. For me, I prefer Z shell, but officially bash is used.

**Comment by ialbert** [comment]
entrez direct changes very frequently, and the bioconda version cannot keep up with it, so bioconda typically will typically produce a much older version

The installer automatically installs using the first method

**Reply by TullyMonster** [reply]
I see. Thank you.
